src/components/organisms/NewProfileModal/NewProfileModalBody/NewProfileModalBody.tsx

```diff
@@ -41,8 +41,8 @@ export const NewProfileModalBody: React.FC<NewProfileModalBodyProps> = ({
     await firebase.auth().signOut();
     closeUserProfileModal();
 
-    history.push(venue?.id ? venueLandingUrl(venue.id) : "/");
-  }, [closeUserProfileModal, firebase, history, venue?.id]);
+    history.push(venue?.slug ? venueLandingUrl(venue.slug) : "/");
```

Comment:
[NOTE[
Instead doing this verbose three operand dance each time `venueLandingUrl()` (and similar functions) is called, they should be made accepting the undefined and add that `/` appropriately.